tutorials-v5/time-evolution/013_nonmarkovian_monte_carlo.md

```diff
@@ -71,24 +85,10 @@ S(t) &= -2 \operatorname{Im} \frac{\dot{c}(t)}{c(t)} \quad \text{and} \\
 
 
 
-##### Imports
-<!-- #endregion -->
-
-```python
-import matplotlib.pyplot as plt
-from matplotlib.collections import LineCollection
-from matplotlib.colors import LogNorm
-
-import numpy as np
-import qutip as qt
-
-from scipy import special, optimize
-from scipy.interpolate import CubicSpline
-```
-
 ##### Setup system
 
 We choose the model parameters $\delta$ and $\beta$:
+<!-- #endregion -->
```

Comment:
What are all the `region` and `endregion` comments for?

Reply:
They were automatically generated by jupyter / jupytext and seem to be surrounding some of the markdown cells. I imagine this might be related to there being code blocks inside some of those markdown cells.

Reply:
That makes sense.